lib/explorer/polars_backend/series.ex

```diff
@@ -117,6 +117,16 @@ defmodule Explorer.PolarsBackend.Series do
   @impl true
   def at(series, idx), do: Shared.apply_series(series, :s_at, [idx])
 
+  @impl true
+  def format(list) do
+    polars_series = for s <- list, do: s.data
+
+    case Explorer.PolarsBackend.Native.s_format(polars_series) do
+      {:ok, %__MODULE__{} = new_series} -> Shared.create_series(new_series)
+      {:error, error} -> raise "Cannot concat series with Polars. Reason: #{inspect(error)}"
```

Comment:
```suggestion
      {:error, error} -> raise "Cannot format series with Polars. Reason: #{inspect(error)}"
```

Reply:
Messages start with lowercase. We should also fix this in concat:

```suggestion
      {:error, error} -> raise "cannot format series with Polars, reason: #{inspect(error)}"
```

Reply:
https://github.com/elixir-nx/explorer/pull/520/commits/90b72f3471805a3ec5029c8adf90c79e62781c90